bio_hansel/main.py

```diff
@@ -246,8 +251,11 @@ def main():
 
     if output_tile_results:
         if len(dfs) > 0:
-            dfall = pd.concat(dfs)  # type: pd.DataFrame
-            dfall = dfall.sort_values(by='is_pos_tile', ascending=False)
+            dfall = pd.concat(dfs, sort=True)  # type: pd.DataFrame
+            if directory_path == None:
+                dfall = dfall.sort_values(by='is_pos_tile', ascending=False)
+            if dfall['subtype'].isnull().any():
+                dfall['subtype'].fillna(value='#N/A', inplace=True)
```

Comment:
See above comment about dropping `.isnull().any()` check.

Reply:
Same as above, that works as well